src/resolve-ref.ts

```diff
@@ -84,32 +87,37 @@ export async function createRefIndex(gitdir: string) {
         let refs = listRefsCache.get(prefix);
 
         if (refs === undefined) {
-            const packedRefs = await readPackedRefs(gitdir);
```

Comment:
These changes are not needed, since `readPackedRefs()` checks a path existence and returns an empty map when it doesn't exist.

Reply:
I will double-check it, but it did explode on the fixture

Reply:
It shouldn't explode for sire since `packed-refs` is optional feature

Reply:
alright, so the problem boils down to the following: now `listRefs` is used inside `listBranches` and `listTags`

`listTags` would explode if `tags` directory is missing, which is exactly the case for current fixture (it's a pain to preserve empty folder in git structure)

Speaking of the `listBranches` , it's available as a part of the `gitReader` API, meaning that consumers could pass any remote name as an argument which would result in an undescriptive error